ci-operator/config/openshift-kni/eco-ci-cd/openshift-kni-eco-ci-cd-main__cnf-core-4.16.yaml

```diff
@@ -0,0 +1,85 @@
+        OCP_MAJOR_VERSION: 4
+        OCP_MINOR_VERSION: 16
+        BASE_DOMAIN: "lab.eng.tlv2.redhat.com"
+        PREDEPLOY_VALIDATE: "false"
+        BAREMETAL_NET_IPV6: "false"
+        BAREMETAL_NET_IPV4: "true"
+        PROVISIONING_NET_IPV6: "false"
+        PROVISIONING_NETWORK_STATE: "Disabled"
+        MANAGEMENT_INTERFACE_TYPE: "RedFish-VirtualMedia"
+        DISCONNECTED_INSTALL: "true"
+        WORKER_ROOT_DISK_HINT_VALUE: "/dev/sda"
+        BYPASS_OLM_SYNC: "true"
+        OLM_USE_NEXUS: "true"
+        OLM_CATALOG_SOURCE: "redhat-operators-brew"
+        OPERATORS_LIST: "ptp-operator, sriov-network-operator, metallb-operator, sriov-fec-operator, kubernetes-nmstate-operator"
+        OLM_ANSIBLE_EXTRA_ARGS: |
+          ocp_dev_version: "4.16"
+          version_intel_acc100: "2.7.2"
+          ose_sriov_network_device_plugin_version: "4.14"
+        CNF_TEST_IMAGE: "registry-proxy.engineering.redhat.com/rh-osbs/openshift4-cnf-tests:v4.16.0-1"
+        DPDK_TEST_IMAGE: "registry-proxy.engineering.redhat.com/rh-osbs/dpdk-base:v4.16.0-1"
+        DISCOVERY_MODE: "false"
+        CNF_POLARION_REPORTING: "false"
+        CNF_TEST_FEATURES_LIST: "sctp,sriov,dpdk,vrf,ovs_qos,metallb,fec,multinetworkpolicy"
+        SCTPTEST_HAS_NON_CNF_WORKERS: "false"
+        CNF_NODES_NUMBER: "3"
+        JENKINS_AGENT: "registry.hlxcl14.lab.eng.tlv2.redhat.com"
+        GINKGO_ARGS: |
+          -e ECO_TEST_FEATURES=core
+          -e ECO_TEST_LABELS=net
+          -e ECO_TEST_VERBOSE='true'
+          -e ECO_VERBOSE_LEVEL=100
+          -e ECO_DUMP_FAILED_TESTS=true
+          -e ECO_REPORTS_DUMP_DIR=/tmp/reports
+          -e ECO_CNF_CORE_NET_VLAN=116
+          -e ECO_CNF_CORE_NET_SRIOV_INTERFACE_LIST='ens3f0np0,ens3f1np1'
+          -e ECO_CNF_CORE_NET_MLB_ADDR_LIST='10.46.186.109,10.46.186.110,10.46.186.111,2620:52:0:2e51::109,2620:52:0:2e51::110,2620:52:0:2e51::111'
+          -e ECO_WORKER_LABEL=workercnf
+          -e ECO_CNF_CORE_NET_TEST_CONTAINER=quay.io/ocp-edge-qe/eco-gotests-network-client@sha256:d4e52aa43f3095e3ee5cf216afedc79997082924d497de14458367af370f4d73
+          -e ECO_CNF_CORE_NET_DPDK_TEST_CONTAINER=quay.io/ocp-edge-qe/eco-gotests-rootless-dpdk@sha256:f35513c7c0778779e2cc71745070e50093a05f22355dfac9bf312bdf6bf22310
+          -e ECO_CNF_CORE_NET_FRR_IMAGE=quay.io/ocp-edge-qe/frr@sha256:e0d0cdc2012f93cf21260d0be6f6c5857523b9649df3c487e63980b564b8e4b3
+          -e ECO_CNF_CORE_NET_SWITCH_IP="10.46.61.153"
+          -e ECO_CNF_CORE_NET_SWITCH_INTERFACES='et-1/0/10,et-1/0/11,et-1/0/12,et-1/0/13'
+          -e ECO_CNF_CORE_NET_SWITCH_USER="root"
+          -e ECO_CNF_CORE_NET_SWITCH_PASS='Qum!8netd'
+          --timeout=12h
+        UPLOAD_REPORTS: "true"
+        FULL_VERSION: "4.16.3"
+      test:
+        - ref: telcov10n-functional-ocp-cnf-tests
+        - ref: telcov10n-functional-ocp-eco-gotests
+        - ref: telcov10n-functional-ocp-cnf-core-tests
+      workflow: telcov10n-functional-cnf-core
+zz_generated_metadata:
```

Comment:
Hi, 
This block is generated automatically for you when you successfully run `make update`, please remove it,

Reply:
removed